EasyPost.Tests/UserTest.cs

```diff
@@ -18,6 +19,24 @@ public void TestRetrieveSelf()
             Assert.AreEqual(user.id, user2.id);
         }
 
+        [TestMethod]
+        public void TestUpdateBrand()
+        {
+            User user = User.Retrieve();
+            Assert.IsNotNull(user.id);
+
+            const string color = "#000000";
+            Dictionary<string, object> details = new Dictionary<string, object>()
+            {
+                { "color", color }
+            };
+            Brand brand = user.UpdateBrand(parameters: details);
+            Assert.IsNotNull(brand);
+            Assert.IsNotNull(brand.user_id);
```

Comment:
This second assertion is inherently useless as we already asserted the object isn't null - objects will always come back with IDs on them.

Reply:
Resolved in [8d459a7](https://github.com/EasyPost/easypost-csharp/pull/203/commits/8d459a71ab9649ee007be4f7891e1ca9c06b371e)